src/router/index.ts

```diff
@@ -35,4 +36,17 @@ const router = createRouter({
   ]
 })
 
-export default router
+
+
+router.beforeEach((to, from, next) => {
+  if (to.name !== "home") {
+    triggerRainingEffect();
+    setTimeout(() => {
```

Comment:
I like to think that setTimeouts are almost always cheating. We'll work on this next week.